components/kubeflow/pytorch-launcher/sample.py

```diff
@@ -66,124 +103,96 @@ def create_worker_spec(
             },
         }
 
-    worker_spec_output = namedtuple(
-        "MyWorkerOutput", ["worker_spec"]
+    return worker
+
+# container component description setting inputs and implementation
+@dsl.container_component
+def pytorch_job_launcher(
```

Comment:
I'd imagine code like would be inside the /src directory instead of sample.py file but I am not familiar enough with KFP codebase to support that.

Reply:
This sample implementation of a pipeline, uses the image in the pytorch job container and runs launch_pytorchjob.py from the /src directory using the parameters etc that are set out, afaiu.

Reply:
yes that is correct